lib/gh/gh.mli

```diff
@@ -5,3 +5,6 @@ module Issue = Issue
 
 (** Generic client to submit GraphQL queries to GitHub API. *)
 module Client = Client
+
+(** Queries and types related to pull requests. *)
+module Pr = Pr
```

Comment:
_Nit:_ Let's move this `Pr` module section right after `Issue`.

I just want `Client` to be at the bottom since it's kinda the last resort that should be used externally